ipynb/diversity-analysis/1-Summary-Table.md

```diff
@@ -0,0 +1,167 @@
+
+The goal of this notebook is to provide demographic summaries for participants in the American Gut and associated projects. We look at metadata, and summarize the available information.
+
+The information generated here will be used for tables 1 of the American Gut paper.
+
+We'll start by importing the necessary libraries.
+
+
+```python
+import matplotlib
+matplotlib.use('Agg')
+
+import os
+
+import numpy as np
+import pandas as pd
+```
+
+Next, we're going to locate the mapping file and raw OTU table.
+
+
+```python
+processing_dir = os.path.abspath('../primary-processing/agp_processing/')
+map_fp = os.path.join(processing_dir, '01-raw/metadata.txt')
+otu_fp = os.path.join(processing_dir, '03-otus/100nt/gg-13_8-97-percent/otu_table.biom')
+```
+
+We'll start by reading in the mapping file.
+
+
+```python
+md = pd.read_csv(map_fp, sep='\t', dtype=str, na_values=['NA', 'no_data', 'unknown', 'Unspecified', 'Unknown'])
+md.set_index('#SampleID', inplace=True)
+```
+
+We're also going to calculate the number of sequences per sample, and add that to our mapping file.
+
+
+```python
+otu_summary = !biom summarize-table -i $otu_fp
+seq_depth = pd.DataFrame(np.array([l.split(': ') for l in otu_summary[15:]]), columns=['#SampleID', 'counts'])
```

Comment:
What about using the [method](https://github.com/biocore/biom-format/blob/master/biom/util.py#L311) directly? It would require loading the table, but the has the advantage of being invariant to the format of the output of `biom summarize-table`